bin/ghe-backup

```diff
@@ -41,8 +41,16 @@ done
 # shellcheck source=share/github-backup-utils/ghe-backup-config
 . "$( dirname "${BASH_SOURCE[0]}" )/../share/github-backup-utils/ghe-backup-config"
 
+# Check that the parallel_command is pointing to moreutils parallel
+if ! $parallel_command -h | grep "parallel \[OPTIONS\] command -- arguments" &>/dev/null; then
+  echo "Error: moreutils parallel not found. Please install https://joeyh.name/code/moreutils" 1>&2
+  exit 1
+fi
+
 # Used to record failed backup steps
 failures=
+failures_path="failures.txt"
```

Comment:
This is making an assumption that this dir is writeable by the backup user as we've not yet changed to the snapshot dir, which won't be the case on those systems with clear separation of responsibility - ie the backup user may not necessarily be the user who installs and updates backup-utils. 

As this is only a temporary file, it's better to use a tmp location using `mktemp` and clean up the file at the end.

Reply:
Fixed